CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Version 3.1.1 (In development)
 
+### App Center
+
+* **[Improvement]** Use standartised `NSUserDefaults` keys
```

Comment:
Can it be breaking change? Dot at the end, add more details about prefix etc

Reply:
Why it's a breaking change if we have migration?

Reply:
Dot at the end, add more details about prefix etc